gitleaks/meta.yaml

```diff
@@ -1,19 +1,18 @@
 {% set name = "gitleaks" %}
-{% set version = "8.8.4" %}
+{% set version = "8.18.4" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  # bash for getting the sigs:
-  # ❯ (GITLEAKS_VER=8.8.4; for _arch in darwin_x64 linux_x64; do curl -sSL https://github.com/zricethezav/gitleaks/releases/download/v${GITLEAKS_VER}/gitleaks_${GITLEAKS_VER}_${_arch}.tar.gz | sha256sum; done)
+  # get the checksums from the checksum file, included in the release
   url: https://github.com/zricethezav/gitleaks/releases/download/v{{ version }}/gitleaks_{{ version }}_darwin_x64.tar.gz # [osx and x86_64]
-  sha256: 509430dada69ee4314068847a8a424d4102defc23fd5714330d36366796feef7 # [osx and x86_64]
+  sha256: 1a69e5666b13cd374889cbcb1939ed1573b63b551251283d5d2329a53cf58e2f # [osx and x86_64]
   url: https://github.com/zricethezav/gitleaks/releases/download/v{{ version }}/gitleaks_{{ version }}_darwin_arm64.tar.gz # [osx and arm64]
-  sha256: 205dfb78a2d8b6ef6837f73892a7c4ca92f02809e70b725ec50cefaab4c4ae29 # [osx and arm64]
+  sha256: a480d8593acd8215b22402cf0f3f88b01dcd3610c63b5391db640f7767e62104 # [osx and arm64]
```

Comment:
Can I get a reviewer to build for macos and push?

(afaik I can't do this myself, but please let me know if this is incorrect)

Reply:
You should be able to with the GH action; see the readme in this repo.

I'll come back to this next week otherwise

Reply:
Got it! I got scared off by this comment: https://github.com/memfault/conda-recipes#building-via-github-action

> Note: Github actions cannot be run on Apple ARM VMs so building the package for darwin-aarch64 still needs to be done "manually"...

Reply:
I ran the Github action but one of the jobs failed and I don't see where I can download any artifacts

![image](https://github.com/user-attachments/assets/c5b83655-e17d-43e1-b690-9cfe72144aed)
https://github.com/memfault/conda-recipes/actions/runs/10445554893

Reply:
I'm not quite sure how this works -- there's new versions for 2 out of the 3 packages on Anaconda.org. Should we have packages for all 3? Do I need a different mac person to build the third?

![image](https://github.com/user-attachments/assets/00ca04f3-cbc1-4a1c-8268-009698b24ded)
https://anaconda.org/memfault/gitleaks

Reply:
I was able to use the Github Actions runner to build artifacts for all macos versions. I'll merge this soon.

Reply:
I confirmed we have all versions